lib/data/marriage_abroad_services.yml

```diff
@@ -575,6 +575,17 @@ romania:
       - :receiving_notice_of_marriage
       - :issuing_cni_or_nulla_osta
   payment_partial_name: pay_by_cash_or_credit_card_no_cheque
+  same_sex:
+    default:
+      partner_british:
+      - :receiving_notice_of_registration
+      - :registering_marriage
+      - :issuing_marriage_certificate
+      partner_other:
+      - :receiving_notice_of_registration
+      - :registering_marriage
+      - :issuing_marriage_certificate
+  payment_partial_name: pay_by_cash_or_credit_card_no_cheque
```

Comment:
partner_british and partner_other have the same services. Can the services just be grouped under default?